src/hotspot/share/utilities/ostream.hpp

```diff
@@ -186,14 +187,13 @@ class streamIndentor : public StackObj {
   ~streamIndentor() { _str->dec(_amount); }
 };
 
-class StreamAutoIndentor : public StackObj {
-  outputStream* const _os;
+class StreamAutoIndentor : public streamIndentor {
```

Comment:
Just a note, not a request for change: I would personally have used composition instead of inheritance here (of `streamIndentor`).